src/routes.js

```diff
@@ -11,13 +11,20 @@ Router.route("/", {
     title: "Home"
 });
 
+Router.route("/room/:_roomNumber/export", {
+    name:"Export",
+    path:"/room/:_roomNumber/export",
+    template: "exportRoom",
+    title: "Export"
+});
```

Comment:
Should probably have the same waitOn and onBeforeAction as the regular room route